models/zuora/zuora_subscription.sql

```diff
@@ -1,12 +1,158 @@
+with sub_mrr as (
```

Comment:
Little bit confused here @3mei . This SQL is for the creation of the model. It isn't an analysis at all.